Spec/Tests/RestClientChannelTests.swift

```diff
@@ -1488,4 +1488,26 @@ class RestClientChannelTests: XCTestCase {
             }
         }
     }
+    
+    // ??
+    func test__047__lifecycle_status_response_should_contain_status_object() {
+        let channelName = uniqueChannelName()
+        let options = AblyTests.commonAppSetup()
+        let testToken = getTestToken(capability: "{\"*\":[\"channel-metadata\"]}")
+        options.token = testToken
+        let client = ARTRest(options: options)
+        let channel = client.channels.get(channelName)
+        
+        waitUntil(timeout: testTimeout) { done in
```

Comment:
Can we test the values inside the `details` object? To do so might require a better understanding of what the values inside the `ARTChannelDetails` object mean (e.g. how do we make `presenceConnections` be a non-zero value etc) which I think will mean looking into the documentation for the API.

Also we might want to test the cases where `status` and `occupancy` (are / are not) null.

Reply:
As [this comment](https://github.com/ably/docs/pull/1455#issuecomment-1153664273) says, these fields should never be nil.
https://github.com/ably/ably-cocoa/pull/1420/commits/8c4610825ced2c1c6d51c19879ea9f46d1ee977e
Test with counters check: https://github.com/ably/ably-cocoa/pull/1420/commits/ab35c8bf3a2931fdbee4f049d695b10ba33b70fb